package.json

```diff
@@ -27,7 +27,16 @@
     "cli-cursor": "^1.0.1",
     "cli-width": "^2.0.0",
     "figures": "^1.3.5",
-    "lodash": "^4.3.0",
+    "lodash.assign": "^4.0.3",
+    "lodash.clone": "^4.3.0",
+    "lodash.defaults": "^4.0.1",
+    "lodash.filter": "^4.2.0",
+    "lodash.flatten": "^4.1.0",
+    "lodash.isarray": "^4.0.0",
+    "lodash.isplainobject": "^4.0.2",
+    "lodash.last": "^3.0.0",
+    "lodash.map": "^4.2.0",
+    "lodash.uniq": "^4.2.0",
```

Comment:
Did you add up the combined package size of each of these modules? The problem as stated earlier is that, inflated with licenses, markdown, etc. these come out to the same size as lodash as a whole.